client/core/core.go

```diff
@@ -2675,6 +2676,100 @@ func (c *Core) Trade(pw []byte, form *TradeForm) (*Order, error) {
 	return corder, nil
 }
 
+// AccountExport is used to retrieve account by host for export.
+func (c *Core) AccountExport(pw []byte, host string) (*Account, error) {
+	_, err := c.encryptionKey(pw)
+	if err != nil {
+		return nil, codedError(passwordErr, err)
+	}
+	_, err = addrHost(host)
+	if err != nil {
+		return nil, newError(addressParseErr, "error parsing address: %v", err)
+	}
+
+	// Get the dexConnection and the dex.Asset for each asset.
+	c.connMtx.RLock()
+	dc, found := c.conns[host]
+	c.connMtx.RUnlock()
+	if !found {
+		return nil, newError(unknownDEXErr, "DEX: %s", host)
+	}
+
+	dc.acct.keyMtx.RLock()
+	accountId := dc.acct.id.String()
+	privKey := hex.EncodeToString(dc.acct.privKey.Serialize())
+	dc.acct.keyMtx.RUnlock()
+
+	// Account ID is exported for informational purposes only, it is not used during import.
+	acct := &Account{
+		Host:      host,
+		AccountID: accountId,
+		PrivKey:   privKey,
+		DEXPubKey: hex.EncodeToString(dc.acct.dexPubKey.SerializeCompressed()),
+		Cert:      hex.EncodeToString(dc.acct.cert),
+		FeeCoin:   hex.EncodeToString(dc.acct.feeCoin),
+		IsPaid:    dc.acct.feePaid(),
+	}
+	return acct, nil
+}
+
+// AccountImport is used import an existing account into the db.
+func (c *Core) AccountImport(pw []byte, acct Account) error {
+	crypter, err := c.encryptionKey(pw)
+	if err != nil {
+		return codedError(passwordErr, err)
+	}
+
+	_, err = addrHost(acct.Host)
+	if err != nil {
+		return newError(addressParseErr, "error parsing address: %v", err)
+	}
+
+	var accountInfo db.AccountInfo
+	accountInfo.Host = acct.Host
+
+	pubKey, err := hex.DecodeString(acct.DEXPubKey)
+	if err != nil {
+		return codedError(decodeErr, err)
+	}
+	accountInfo.DEXPubKey, err = secp256k1.ParsePubKey(pubKey)
+	if err != nil {
+		return codedError(parseKeyErr, err)
+	}
+
+	accountInfo.Cert, err = hex.DecodeString(acct.Cert)
+	if err != nil {
+		return codedError(decodeErr, err)
+	}
+
+	accountInfo.FeeCoin, err = hex.DecodeString(acct.FeeCoin)
+	if err != nil {
+		return codedError(decodeErr, err)
+	}
+
+	accountInfo.Paid = acct.IsPaid
```

Comment:
Well, I had not thought about this, but the `Paid` part is not encoded in the same DB bucket with the `AccountInfo`
See `(db *BoltDB).AccountPaid` and `feeProofKey`.  I think we're going to need to backup the `AccountProof` too.

```go
type AccountProof struct {
	Host  string
	Stamp uint64
	Sig   []byte
}
```

The new account struct already has a `Host string`, so maybe the exported struct with json tags can have an `FeeProofSig` and `FeeProofStamp`.  This means on import using the `AccountPaid` db method to store the loaded fee proof, and for export creating a new BoltDB method to load the AccountProof since we don't have one.

Reply:
Maybe something like the following to load the `AccountProof` for export:

```go
func (db *BoltDB) AccountProof(url string) (*dexdb.AccountProof, error) {
	var acctProof *dexdb.AccountProof
	acctKey := []byte(url)
	return acctProof, db.acctsView(func(accts *bbolt.Bucket) error {
		acct := accts.Bucket(acctKey)
		if acct == nil {
			return fmt.Errorf("account not found for %s", url)
		}
		var err error
		acctProof, err = dexdb.DecodeAccountProof(getCopy(acct, feeProofKey))
		if err != nil {
			return err
		}
		return nil
	})
}
```

For import there's the existing `(db *BoltDB) AccountPaid(proof *dexdb.AccountProof)`